hawc/apps/animal/views.py

```diff
@@ -201,6 +201,7 @@ def get_context_data(self, **kwargs):
         )
         context["config"] = dict(
             id=self.object.id,
+            has_dosing_regime=self.object.dosing_regime is not None,
```

Comment:
changed to set this variable in the config, rather than the html template, it should be a lot cleaner this way and is more like our "normal" path that has slowly evolved